### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by e-jim**
Hello,

I added some new translations + corrected some existing ones.

I hope this helps.

Thanks for your job on this GREAT extension.

### CODE REVIEW

File: src/locale/fr/requestpolicy.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
-extensionConflictWarning=Conflicting extensions: %S
-noOrigin=[no origin]
+extensionConflictWarning=Extension en conflit: %S
```
Comment (nodiscc) [inline src/locale/fr/requestpolicy.properties]: Extensions

File: src/locale/fr/requestpolicy.properties [hunk 2]
```diff
@@ -76,25 +76,25 @@ addRule=Ajouter une règle
...
+manageSubscriptions=Gérer les souscriptions
+subscriptionPolicies=Politiques issues des souscriptions
+subscriptionPoliciesDefinition=Les politiques issues des souscriptions sont des règles prédéfinies, maintenues par la communauté. Ces règles sont automatiquement mises à jour et peuvent répondre à différents besoins, tels que le blocage des sites connus pour le traçage de votre navigation ou l'autorisation de requêtes permettant à certains sites de fonctionner ou de s'afficher normalement. Les issues des souscriptions fonctionnent comme des règles additionnelles par rapport à celles définies par l'utilisateur, c'est à dire que si aucune règle définie par l'utilisateur ne s'applique, RequestPolicy vérifira si une subscription est applicable.
+learnMoreAboutSubscriptions=En savoir plus sur les souscriptions.
+privacy=Vie privée
+usability=Facilité d'utilisation
+browser=Navigateur
+subscriptionDenyTrackersDescription=Bloque les destinations qui peuvent vous pister lors de votre navigation sur le web
+subscriptionAllowSameOrgDescription=Accepte les destination qui font partie de la même organisation que le site d'origine
```
Comment (nodiscc) [inline src/locale/fr/requestpolicy.properties]: destinations

File: src/locale/fr/requestpolicy.properties [hunk 3]
```diff
@@ -102,24 +102,24 @@ returnToBrowsing=Retour à la navigation
...
-showOldRuleReimportOptions=Show re-import options.
-yourOldRulesHaveBeenDeleted=Done! Your old rules have been deleted.
-ruleSet=Rule Set
-filterRules=Filter Rules
-policy=Policy
+removeOldRules=Supprimer les règles non-utilisées issues de l'ancienne version de RequestPolicy.
+importOldRules=Importer d'anciennes règles
+deleteOldRules=Supprimer les anciennes règles
+showOldRuleReimportOptions=Montrer les options de ré-import.
+yourOldRulesHaveBeenDeleted=Voilà qui fait! Vos anciennes règles ont été effacées.
```
Comment (nodiscc) [inline src/locale/fr/requestpolicy.properties]: Voilà qui est fait

**Comment by nodiscc** [comment]
Thank you et merci. J'ai ajouté des notes sur les quelques typos qui restent. @e-jim peux tu les corriger?

Let me know if you need help

**Comment by e-jim** [comment]
There it is. Thanks for the thorough review. I'll be more vigilant next time.

**Comment by nodiscc** [comment]
thanks, @myrdd good to merge

**Comment by myrdd** [comment]
Thank you @e-jim

**Pull request closed without merging**